.travis.yml

```diff
@@ -21,7 +21,8 @@ env:
     # Set default python version to avoid repetition later
     - BUILD_DOCS=false
     - PYTHON_VERSION=2.7
-    - MAIN_CMD="python ./testsuite/MDAnalysisTests/mda_nosetests --processes=2 --process-timeout=400 --no-open-files --with-timer --timer-top-n 50"
+    - MAIN_CMD="python ./testsuite/MDAnalysisTests/mda_nosetests analysis auxiliary coordinates core formats topology utils
```

Comment:
One way of making this cleaner, would be to use `./testsuite/MDAnalysisTests/mda_nosetests $(cat tests_for_nose.txt) --options`. This assumes that `test_for_nose.txt` is a file where each line is a module to test. One advantage is that you can have a `tests_for_pytest.txt` so moving tests from one system to the other can be done by moving the corresponding lines from one file to the other. It should look clearer in the diffs, and it will make `.travis.yml` a bit nicer to read.